### ISSUE TITLE
Exception when running slash commands in DM with bot

### ISSUE THREAD

**Issue by XenoTheStrange** [kkrypt0nn/Python-Discord-Bot-Template#61]
As title. This is the exception. It fires on most commands.

Ignoring exception in on_slash_command
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/disnake/client.py", line 612, in _run_event
    await coro(*args, **kwargs)
  File "./bot.py", line 147, in on_slash_command
    f"Executed {interaction.data.name} command in {interaction.guild.name} (ID: {interaction.guild.id}) by {interaction.author} (ID: {interaction.author.id})")
AttributeError: 'NoneType' object has no attribute 'name'

This error does not appear in channels. The f-string renders properly there.
So... either interaction.guild or interaction.data is returning NoneType I guess. Idk.

Good work! This error gets ignored and the bot continues working just fine.

**Comment by kkrypt0nn** [comment]
Hey,

Yeah that is my bad, when initially creating the template with slash commands I thought they were only available in guilds and not private messages. That's why `interaction.guild` will be `None`, there is no guild in the interaction object.

I will edit that and add a simple check to that hopefully in the coming days.

Thanks for pointing that out

**Comment by kkrypt0nn** [comment]
Fixed in commit [e83677d](https://github.com/kkrypt0nn/Python-Discord-Bot-Template/commit/e83677d3d29449e6c8a1a29d9cf0399b3da2d2c1).

It will now display the following in the terminal
```
Executed ping slash command by Krypton#7331 (ID: 562359123086409729) in DMs
Executed ping command by Krypton#7331 (ID: 562359123086409729) in DMs
```
